lib/screens/v2/candidate_generator/elevator/closures.ex

```diff
@@ -0,0 +1,160 @@
+defmodule Screens.V2.CandidateGenerator.Elevator.Closures do
+  @moduledoc false
+
+  require Logger
+
+  alias Screens.Alerts.{Alert, InformedEntity}
+  alias Screens.Facilities.Facility
+  alias Screens.Routes.Route
+  alias Screens.Stops.Stop
+  alias Screens.V2.WidgetInstance.ElevatorClosures
+  alias Screens.V2.WidgetInstance.Serializer.RoutePill
+  alias ScreensConfig.Screen
+  alias ScreensConfig.V2.Elevator
+
+  @stop Application.compile_env(:screens, [__MODULE__, :stop_module], Stop)
+  @facility Application.compile_env(:screens, [__MODULE__, :facility_module], Facility)
+  @alert Application.compile_env(:screens, [__MODULE__, :alert_module], Alert)
+  @route Application.compile_env(:screens, [__MODULE__, :route_module], Route)
+
+  @spec elevator_status_instances(Screen.t()) :: list(ElevatorClosures.t())
+  @spec elevator_status_instances(Screen.t(), DateTime.t()) :: list(ElevatorClosures.t())
+  def elevator_status_instances(
+        %Screen{
+          app_params: %Elevator{
+            elevator_id: elevator_id
+          }
+        },
+        now \\ DateTime.utc_now()
+      ) do
+    with {:ok, %Stop{id: stop_id}} <- @facility.fetch_stop_for_facility(elevator_id),
+         {:ok, location_context} <- @stop.fetch_location_context(Elevator, stop_id, now),
```

Comment:
It seems like the only part of the LocationContext we ultimately use here is the `home_stop`, which is simply the exact `stop_id` we're passing into this function. Can we save some data fetching by omitting this?